### PR TITLE
Conformance access_test

### PR DESCRIPTION
**Pull request by fossedihelm**
### What this PR does
~This PR aims to identify and label the tests in the access_test and scale/virt-api files.~
This PR aims to identify and label the tests in the access_test file.

### Release note

```release-note
NONE
```

### CODE REVIEW

File: tests/scale/virt-api.go [hunk 1]
```diff
@@ -74,7 +74,7 @@ var _ = Describe("[sig-compute] virt-api scaling", decorators.SigCompute, func()
 		return expectedReplicas
 	}
 
-	It("virt-api replicas should be scaled as expected", func() {
+	It("virt-api replicas should be scaled as expected", decorators.Conformance, func() {
```
Comment (xpivarc) [inline tests/scale/virt-api.go:77]: This depends on the env, I think it does not meet criteria

File: tests/access_test.go [hunk 2]
```diff
@@ -215,7 +214,7 @@ var _ = Describe("[rfe_id:500][crit:high][arm64][vendor:cnv-qe@redhat.com][level
 	})
 
 	Describe("With default kubevirt service accounts", func() {
-		DescribeTable("should verify permissions on resources are correct for view, edit, and admin", func(group string, resource string, clusterWide bool, accessRights ...rights) {
+		DescribeTable("should verify permissions on resources are correct for view, edit, and admin", decorators.Conformance, func(group string, resource string, clusterWide bool, accessRights ...rights) {
```
Comment (xpivarc) [inline tests/access_test.go:217]: I am little bit torn here...

File: tests/scale/virt-api.go (hunk 1 above)
Comment (fossedihelm) [reply tests/scale/virt-api.go:77]: Oh, that's right! Nice catch!

File: tests/access_test.go (hunk 2 above)
Comment (fossedihelm) [reply tests/access_test.go:217]: Can you elaborate? :)

Comment (xpivarc) [reply tests/access_test.go:217]: I think different deployments might want to have RBAC set up differently. Also the RBAC wasn't or might still not be on by default on Kubernetes side.

**Pull request closed without merging**